index.html

```diff
@@ -89,7 +89,7 @@
     <div class="intro">
         <img style="margin-bottom: 30px; max-width:90%" src="assets/images/s2s_logotrimmed.png" alt="Logo">
         <h1 style="text-transform: none;">Bridging the gap between students and startups</h1><br>
-        <h2 style="margin-bottom: 30px; font-weight: bold;">Thursday October 19th, 2017 <br> University of Colorado Boulder <br> 6:00 - 8:00 PM </h2>
+        <h2 style="margin-bottom: 30px; font-weight: bold;">Thursday October 19th, 2017 -  6 PM<br> University of Colorado Boulder </h2>
```

Comment:
I think it would look better if they are on separate lines, but one after the other (unlike what we have now)
> Thursday October 19th, 2017
> 6 PM to 8 PM
> University of Colorado Boulder

I think this feels more coherent and complete. What do you think?